docs/csharp/whats-new/csharp-7-1.md

```diff
@@ -31,6 +31,14 @@ control [reference assembly generation](#reference-assembly-generation).
 
 To use the latest features in a point release, you need to [configure the compiler language version](../language-reference/configure-language-version.md) and select the version.
 
+The remainder of this article provides an overview of each feature. For each feature,
+you'll learn the reasoning behind it. You'll learn the syntax. You can explore these features in your environemnt using the `dotnet try` global tool:
+
+1. Install [dotnet-try](https://github.com/dotnet/try/README.md#setup)
```

Comment:
```suggestion
1. Install the [dotnet-try](https://github.com/dotnet/try/blob/master/README.md#setup) global tool.
```

Reply:
I created suggestions for the steps here, but not in the other articles. Since it's just repeating I figured it would be easier if you did it in the source material **IF** you agreed with this these changes. Maybe you don't want the steps altered 😄 

The link though doesn't work so I'll put that in as a suggestion for each entry. Note that the other ones don't have my step description suggestions.